code/game/machinery/shieldgen.dm

```diff
@@ -65,6 +65,7 @@
 	name = "Invoker's Shield"
 	desc = "A weak shield summoned by cultists to protect them while they carry out delicate rituals"
 	color = "red"
+	atom_colours = list("", "", "", "red")
```

Comment:
This isn't your fault but I really wish people would use actual colors rather than, uh, whatever color code this turns out to be?